### DISCUSSION TITLE
Plugin: 0-255 RGB Values

### DISCUSSION THREAD

**Discussion by lajbel** [replit/kaboom#137] [Show and tell]
```.js
kaboom({
    global: true,
    plugins: [realRgbPlugin]
});

obj = add([
     rect(50, 50),
     color(realRgb(255, 218, 115)), // returns rgba(1, 0.854..., 0.607..., 1);
     // color(realRgb(255, 218, 115, 0.5)) // can modify the alpha values
]);
```
I have made this plugin for myself and anyone who wants to use it, since it is faster than having to manually divide your rgb values ​​or having to add more code, a simple plugin.

Plugin code (and import if you want) https://lajbel.repl.co/plugins/realRGB.js

**Comment by slmjkdbtl** [comment]
Nice. If most people really prefer using 0-255 I'm also interested making a global flag for that (will require some refactor tho)
